src/us/kbase/auth2/lib/identity/IdentityProviderConfig.java

```diff
@@ -76,20 +85,53 @@ public String getClientID() {
 	public String getClientSecret() {
 		return clientSecret;
 	}
+	
+	/** Get the environments in the configuration.
+	 * See {@link Builder#withEnvironment(String, URL, URL)}.
+	 * @return the environements.
+	 */
+	public Set<String> getEnvironments() {
+		return envLoginRedirectURL.keySet();
+	}
 
-	/** Get the URL to which the identity provider should redirect when a login is in process.
+	/** Get the default URL to which the identity provider should redirect when a login is in
+	 * process.
 	 * @return the login redirect url.
 	 */
 	public URL getLoginRedirectURL() {
```

Comment:
Should the method be renamed too?

Reply:
Nah, the lack of a parameter implies default IMO.